library/alloc/src/fmt.rs

```diff
@@ -12,19 +12,19 @@
 //! Some examples of the [`format!`] extension are:
 //!
 //! ```
-//! format!("Hello");                 // => "Hello"
-//! format!("Hello, {}!", "world");   // => "Hello, world!"
-//! format!("The number is {}", 1);   // => "The number is 1"
-//! format!("{:?}", (3, 4));          // => "(3, 4)"
-//! format!("{value}", value=4);      // => "4"
+//! let _ = format!("Hello");                 // => "Hello"
+//! let _ = format!("Hello, {}!", "world");   // => "Hello, world!"
+//! let _ = format!("The number is {}", 1);   // => "The number is 1"
+//! let _ = format!("{:?}", (3, 4));          // => "(3, 4)"
+//! let _ = format!("{value}", value=4);      // => "4"
 //! let people = "Rustaceans";
-//! format!("Hello {people}!");       // => "Hello Rustaceans!"
-//! format!("{} {}", 1, 2);           // => "1 2"
-//! format!("{:04}", 42);             // => "0042" with leading zeros
-//! format!("{:#?}", (100, 200));     // => "(
-//!                                   //       100,
-//!                                   //       200,
-//!                                   //     )"
+//! let _ = format!("Hello {people}!");       // => "Hello Rustaceans!"
+//! let _ = format!("{} {}", 1, 2);           // => "1 2"
+//! let _ = format!("{:04}", 42);             // => "0042" with leading zeros
+//! let _ = format!("{:#?}", (100, 200));     // => "(
+//!                                           //       100,
+//!                                           //       200,
+//!                                           //     )"
```

Comment:
Imo adding the `let _ =` is just some unnecessary noise in a commonly referenced module. Could we just `println!` instead? Or disable the warning in these docs?

Reply:
this example is about `format!` so we should be using it and not `println!`.
But we can disable the warning in this doc